src/battle_ai_util.c

```diff
@@ -3394,13 +3420,9 @@ bool32 PartnerMoveIsSameNoTarget(u32 battlerAtkPartner, u32 move, u32 partnerMov
 
 bool32 PartnerMoveActivatesSleepClause(u32 partnerMove)
 {
-    u32 effect = gMovesInfo[partnerMove].effect;
     if (!IsDoubleBattle() || !FlagGet(B_FLAG_SLEEP_CLAUSE))
         return FALSE;
-    if (effect == EFFECT_SLEEP
-        || effect == EFFECT_YAWN)
-        return TRUE;
-    return FALSE;
+    return IsMoveSleepClauseTrigger(partnerMove);
```

Comment:
This function just calls the other function, right? Any reason the could would not just be here?

Reply:
not blocking, just curious

Reply:
My thought process is entirely futureproofing, if we or anyone downstream ends up wanting to do anything funky with sleep clause and the AI this helper function can just be used to grab every effect they'd want to consider